api/src/main/java/org/pmiops/workbench/cdr/dao/DbDomainDao.java

```diff
@@ -0,0 +1,10 @@
+package org.pmiops.workbench.cdr.dao;
+import org.pmiops.workbench.cdr.model.DbDomain;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface DbDomainDao extends CrudRepository<DbDomain, Long> {
+
+    List<DbDomain> findAllByDomainIdIsNotNull();
```

Comment:
Can we write unit test for this Dao?

Reply:
Our general policy is to write unit tests for significant code you check in. Otherwise somebody else is going to break your code and we won't know it. So if it's not going to be in this PR, let's do it soon?